test/controllers/relationships_controller_test.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class RelationshipsControllerTest < ActionDispatch::IntegrationTest
+  include Devise::Test::IntegrationHelpers
+
+  setup do
+    sign_in users(:one)
+    @user_two = users(:two)
```

Comment:
```suggestion
    @hansen = users(:hansen)
```

oneやtwoより、具体的な名前を使った方がわかりやすいかも？